lib/Doctrine/Migrations/VersionDirection.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Migrations;
+
+final class VersionDirection
+{
+    public const DIRECTION_UP   = 'up';
+    public const DIRECTION_DOWN = 'down';
```

Comment:
`VersionDirection::UP` + `VersionDirection::DOWN` is imho the way we want to go.